docs/Installation.md

```diff
@@ -523,6 +531,10 @@ install -v -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
 install -v -m 755 ./zm_testagent-bsd /usr/local/etc/rc.d/zm_testagent
 ```
 
+> If this is an update of Zonemster-Backend, you should remove any script from older
+> version of Zonemaster-Backend in `/usr/local/etc/rc.d`, i.e. any for Zonemaster except for
+> the ones just installed.
+

```

Comment:
This can be removed. On FreeBSD the names have always been the same.

Reply:
Good suggestion. Will fix.